src/main/kotlin/no/nav/syfo/dialogmelding/database/domain/PMelding.kt

```diff
@@ -17,3 +20,16 @@ data class PMelding(
     val tekst: String?,
     val antallVedlegg: Int,
 )
+
+fun PMelding.toMeldingTilBehandler() = MeldingTilBehandler(
+    uuid = uuid,
+    createdAt = createdAt,
+    type = DialogmeldingType.valueOf(type),
+    conversationRef = conversationRef,
+    parentRef = parentRef,
+    bestiltTidspunkt = tidspunkt,
+    arbeidstakerPersonIdent = PersonIdent(arbeidstakerPersonIdent),
+    behandlerRef = behandlerRef ?: throw IllegalStateException("Mangler behandlerRef for MeldingTilBehandler"),
+    tekst = tekst ?: throw IllegalStateException("Mangler tekst for MeldingTilBehandler"),
```

Comment:
Her får null som må håndteres, er det greit å kaste feil for det?

Reply:
Litt usikker - er jo ikke noe som skal skje normalt. Men kanskje bedre å bruke tom streng som default i stedet for å kaste en exception?

Reply:
Ja, kanskje det 🤔

Reply:
Tom streng som `tekst` er kanskje greit, men det bør vel feile hvis `behandlerref` er null her så da tenker jeg exception blir riktig.

Reply:
Enig i at manglende behandlerref er en feil, og da tenker jeg vi kan reagere på det.